modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+----
++
+For information about configuring this file for your specific content, contact a member of the CCS tooling team.
+
+This repository structure offers the following advantages:
+
+* *Flexibility in determining publishable titles.* In this structure, the images, artifacts, assemblies, and modules all serve as distinct building blocks that you use to compile a document or _title_. You can include an image in multiple modules, a module in multiple assemblies, and an assembly in multiple titles. Additionally, just as an assembly can include multiple modules, a title can contain multiple assemblies.
+* *More intuitive content reuse.* When you decouple the standalone content components from the publishable titles, you avoid potentially awkward includes when modules or assemblies from one title are reused in another. For example, consider a repository where each publishable title folder contains all modules or assemblies or both directly within it. This creates a dependency for that module or assembly content on that title. As you reuse content across title folders, you end up with includes that direct to various other unrelated titles to retrieve the content. Alternatively, in the recommended repository structure, you can reuse all content independent of the titles in which you include it.
+* *Potentially easier collaboration in community and between content repositories.* For repositories that are single-sourced with community projects or that need to reuse content between product documentation repositories, this structure enables you to isolate the enterprise title compilation requirements so that the standalone components can be shared more readily. However, single-sourcing between community and enterprise content repositories poses other challenges beyond the scope of this document. (See the disadvantages and community considerations sections that follow.)
+* *Consistency with Pantheon 2 YAML configurations.* In this structure, the division of modules, assemblies, and other resources is consistent with the division of components in the `pantheon2.yml` file that is required for documents published on Pantheon 2.
+
+This repository structure poses the following disadvantages:
+
+* *Extensive use of symlinks.* In order to enable titles to be built with distinct assemblies, modules, and images, this repository structure uses many symlinks (symbolic links) for access across directories. You can extract these symlinks with a build script or additional variable configurations (such as `imagesdir`), but because this example repository is intended for a general CCS audience, it maintains explicit symlinks so that a user can build the repository titles with standard CCS and Asciidoctor tooling. Alternative repository structures that combine all relevant images, modules, and assemblies together in title folders can reduce the number of symlinks, but these alternative structures then forego many of the advantages listed previously.
```

Comment:
```suggestion
* *Extensive use of symlinks.* In order to enable titles to be built with distinct assemblies, modules, and images, this repository structure uses many symlinks (symbolic links) for access across directories. You can eliminate these symlinks with a build script or additional variable configurations (such as `imagesdir`), but because this example repository is intended for a general audience, it maintains explicit symlinks, so that a user can build the repository titles with standard Asciidoctor tooling. Alternative repository structures that combine all relevant images, modules, and assemblies together in title folders can reduce the number of symlinks, but these alternative structures then forego many of the advantages listed previously.
```